reporting-analysis/load-test/README.md

```diff
@@ -0,0 +1,61 @@
+# Load tests for Payments project
+
+
+- [Load tests for Payments project](#load-tests-for-payments-project)
+  - [01. get flow list call](#01-get-flow-list)
+  - [02. get flow call](#02-get-flow)
+
+This is a set of [k6](https://k6.io) load tests related to the GPD (_Gestione Posizioni Debitorie_) initiative.
+
+To invoke k6 load test passing parameter use -e (or --env) flag:
+
+```
+-e MY_VARIABLE=MY_VALUE
+```
+
+## 01. Get Flow List
+
+Call to test the get flow list call:
+1. initializes the table by inserting test records
+2. execute rest call
+3. clears the table from the test records entered
+
+Example command launch on localhost 
+
+```
+k6 run -e BASE_URL=http://localhost:7071 -e FLOW_TABLE_URL=http://127.0.0.1:10002/devstoreaccount1/flows -e ACCESS_SIGNATURE_QUERY_STRING="?sv=2018-03-28&se=2022-04-28T12%3A15%3A36Z&sp=raud&sig=dwGaSA6cXmAk6ZHPrhrg97up9qmvm0W6IAFUAYfKTdA%3D&tn=flows" -e NUM_OF_ORGANIZATIONS=5 -e NUM_FLOW_ID_FOR_ORG=10 reporting-analysis\load-test\src\reporting-analysis_get_flow_list.js
```

Comment:
add pre-condition section  in the README to say that localhost test makes use of
  https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=docker-hub

Reply:
done